kroxylicious-sample/README.md

```diff
@@ -8,27 +8,31 @@ This sample filter project provides examples to help you learn how [custom filte
 
 Building the sample project is easy! You can build the **kroxylicious-sample** jar either on its own or with the rest of the Kroxylicious project.
 
-To build all of Kroxylicious, including the sample:
+#### To build all of Kroxylicious, including the sample:
 
 ```
-$ mvn clean install
+$ mvn verify --also-make
```

Comment:
I don't think we need `also-make` when building the entire reactor

Reply:
Very true, have fixed now.